brickstrap-argv.sh

```diff
@@ -0,0 +1,237 @@
+#!/bin/bash
+#
+# This file is part of brickstrap.
```

Comment:
Just throwing this out there... Should we be writing brickstrap in python or some other language that has a decent arg parsing library so that we don't have to do all of this ourselves?